src/runtime/RegExpObject.cpp

```diff
@@ -497,4 +497,14 @@ void RegExpObject::pushBackToRegExpMatchedArray(ExecutionState& state, ArrayObje
         }
     }
 }
+
+bool RegExpObject::checkMatchSymbol(ExecutionState& state)
+{
+    ObjectGetResult symbol = get(state, ObjectPropertyName(state, state.context()->vmInstance()->globalSymbols().match));
+    Value val = symbol.value(state, Value());
```

Comment:
The second argument must be `Value(this)`, due to the possible getter function invocation.